### ISSUE TITLE
Adjust Seeders for Hawaii

### ISSUE THREAD

**Issue by andrew-rist-tw** [PublicDataWorks/complaint-manager#223]
~Blocked by Story #226~

### Business Context:
I don't think we want to lose the New Orleans seeders, but we need to be able to run seeders specifically for Hawaii (because custom configuration isn't fully fledged enough yet for Christin to do this on her own).  There are a couple of ways (I think) to adapt a seeder to branch between NOIPM and Hawaii:
1. Get whatever seed data from instance-files (some seeders already do this).  I suppose it is okay to add things to instance-files if this is the best solution (just make sure if we've created hawaii instance files that you add the new vars to both noipm and Hawaii instance-files)
2. use a new environment variable (should be the same variable for all seeders using this strategy), let's call it ORG, to determine which org it's for and seeding appropriately

**As a** person who wants to see what the app will look like for Hawaii
**I want** to be able to flip any environment from NOIPM to Hawaii
**So that** I can see what a change will look like in that setup

### In Scope
1. Alter each seeder to use Hawaii data

### Out of Scope
1. Script to undo one org's seeders and apply another's
3.
4.

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: 
Given
When
Then

2. Scenario: 
Given
When
Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
6. Determine if documentation should be referenced or created.
7. Ensure to include all necessary mockups to this card.
8. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
9. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
10. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
11. Have we added edge cases as ACs that we're considered from a QA perspective
12. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**

**Comment by andrew-rist-tw** [comment]
Seeder Documentation: https://sequelize.org/docs/v6/other-topics/migrations/

**Comment by andrew-rist-tw** [comment]
Hawaii Complaint Types:
* Medical
* Mental Health
* Staff Misconduct
* PREA
* Mail
* Food
* Property
* Visitation
* Programming
* Reentry
* Court Case
* Parole
* Probation
* Security Classification
* Facility-Wide
* Department-Wide
